pkg/helpers/newapp/newapptest/newapp_test.go

```diff
@@ -1015,14 +1015,14 @@ func TestNewAppRunBuilds(t *testing.T) {
 			name: "successful build from dockerfile",
 			config: &cmd.AppConfig{
 				GenerationInputs: cmd.GenerationInputs{
-					Dockerfile: "FROM openshift/origin:v1.0.6\nUSER foo",
+					Dockerfile: "FROM centos:centos8\nUSER foo",
 				},
 			},
 			expected: map[string][]string{
-				"buildConfig": {"origin"},
+				"buildConfig": {"centos"},
```

Comment:
I _think_ this will create a `ubi-micro` BuildConfig.

```suggestion
				"buildConfig": {"ubi-micro"},
```